index.html

```diff
@@ -45,30 +45,76 @@ <h1>{A little bit about me...}</h1>
 				<p>Please keep scrolling to see some of my work.</p>
 			</div>
 		</section>
```

Comment:
move this opening section tag and all tags within this section all to the left one tab, so the indentation is all in line (e.g. section to line up with nav)